src/main.c

```diff
@@ -1,3 +1,5 @@
+#include "wren_common.h"
```

Comment:
We definitely don't want this. The only public header Wren exposes is wren.h. Including this leaks implementation details of the VM.